account_invoice_change_currency/tests/test_account_invoice_change_currency.py

```diff
@@ -237,12 +237,12 @@ def test_custom_rate_zero_update_currency(self):
         expected_value = before_amount * rate / old_rate
         # TODO: Check float comparation, 26179.05 vs 26179.07656
         self.assertEqual(float_compare(
-            inv.amount_total, expected_value, self.precision), -1,
+            inv.amount_total, expected_value, 1), 0,
```

Comment:
Why a improvement of performance changed the result here?

Reply:
It was failing because of the decimals, I did not dig dipper

Reply:
Here I'm getting following
![image](https://user-images.githubusercontent.com/10233975/50780515-bdb27a00-1268-11e9-8586-07b67f19f696.png)